core/domain/story_services_test.py

```diff
@@ -159,13 +162,27 @@ def test_update_story_properties(self):
                 'new_value': 'new story meta tag content'
             })
         ]
+
+        # Save a dummy image on filesystem, to be used as thumbnail.
+        with python_utils.open_file(
+            os.path.join(feconf.TESTS_DATA_DIR, 'image.svg'),
+            'rb', encoding=None) as f:
+            raw_image = f.read()
+        fs = fs_domain.AbstractFileSystem(
+            fs_domain.GcsFileSystem(
+                feconf.ENTITY_TYPE_STORY, self.STORY_ID))
+        fs.commit(
+            '%s/image.svg' % (constants.ASSET_TYPE_THUMBNAIL), raw_image,
+            mimetype='image/svg+xml')
+
         story_services.update_story(
             self.USER_ID, self.STORY_ID, changelist,
             'Updated Title and Description.')
         story = story_fetchers.get_story_by_id(self.STORY_ID)
         self.assertEqual(story.title, 'New Title')
         self.assertEqual(story.description, 'New Description')
         self.assertEqual(story.thumbnail_filename, 'image.svg')
+        self.assertEqual(story.thumbnail_size_in_bytes, len(raw_image))
         self.assertEqual(
```

Comment:
@seanlip
Ditto here, do you think we need tests for the case when the thumbnail does not exist in the filesystem ?

Reply:
Don't think you need it here since it's not being handled in story_services (the changes to that file in this PR seem minor). But no harm adding it if it's easy, I guess.